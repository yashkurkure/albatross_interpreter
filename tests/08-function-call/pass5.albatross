fun fv void () {
  return;
}

fv();