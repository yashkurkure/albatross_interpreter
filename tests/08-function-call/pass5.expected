Function declared "fv" returns void
Function called "fv" returns void
